disco_aws_automation/disco_aws.py

```diff
@@ -409,6 +427,11 @@ def provision(self, ami, hostclass=None,
 
         self.create_scaling_schedule(hostclass, min_size, desired_size, max_size)
 
+        # Create alarms and custom metrics for the hostclass, if is not being used for testing
+        if not testing:
+            hostclass_alarms = self.alarm_configs.get_alarms(hostclass, group.name)
+            self.alarms.create_alarms(hostclass_alarms)
```

Comment:
Why not move this logic to disco_alarms.py and keep disco_aws.py oblivious of the implementation of create_alarms?